### DISCUSSION TITLE
How to recover data in a active-active cluster

### DISCUSSION THREAD

**Discussion by BrucePeng92** [minio/minio#16199] [Q&A] [answered]
I have two minio clusters. I configure replication from A to B, and B to A.
I have a question, If cluster A is unavailable, how to recover data from cluster B after the cluster is restored？
Use `mc cp -r ` or `mc mirror`? What is different between two command?
What should I do if there is new data in the copy process?

**Comment by harshavardhana** [comment, marked as answer]
https://min.io/docs/minio/kubernetes/upstream/operations/data-recovery/recover-after-site-failure.html
